### ISSUE TITLE
How to create a new entry as disabled from form?

### ISSUE THREAD

**Issue by fredrik-sogaard** [verbb/formie#622]
**Description**
I'm using the Element integration to save an entry from a form. I want to set the entry as disabled (or draft).
In the Entry fields mapping there is a field for Enabled, and I've tried to map this to a hidden field with values like `true`, `1`,`enabled`, `live` and others. But everything creates an enabled entry.

How does this field work? And can I create disabled entries?

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation:No
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.4.19
- Craft version: 3.7.17.1
- Multi-site: No

**Additional context**
I've asked this question as a support ticket as well, but no reply yet.

**Comment by engram-design** [comment]
Yep, looks like boolean values aren't being treated properly. Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.22",
  "...": "..."
}
```

Then run `composer update`.

As for your support ticket, I'm afraid we haven't received that, so sorry for ignoring you!

**Comment by fredrik-sogaard** [comment]
OK, thanks! But what should the value be? I've updated (https://cln.sh/WBDFVt) and tried `1` and `true` but it still won't work? Am I missing something here?

**Comment by engram-design** [comment]
If you provide a boolean-like value, it should work, as I specifically tested a hidden field with the value `false`.

If you're setting the "Enabled" value to `true` or `1` then it will be enabled. To disable it, use a falsey value like `0` or `false` - that should do the trick, but let me know if not.

**Comment by fredrik-sogaard** [comment]
Of course. What was I thinking. Everything works now! Thanks for your help!

**Comment by engram-design** [comment]
Also just double checked our support tickets, and looks like your ticket was marked as spam, due to failing reCAPTCHA. Just in case you try and get in touch again - but then the site does still use Sprout Forms (for the moment!) which can be prone to reCAPTCHA issues.

**Comment by engram-design** [comment]
Fixed in [1.4.23](https://github.com/verbb/formie/releases/tag/1.4.23)
